tensorboard/webapp/widgets/histogram/histogram_component.ng.html

```diff
@@ -50,14 +50,14 @@
       </g>
     </svg>
     <!-- Disable the feature when in non-offset and non-step mode. -->
-    <ng-container *ngIf="isLinkedTimeEnabled(linkedTime)">
+    <ng-container *ngIf="isTimeSelectionEnabled(timeSelection)">
       <histogram-card-fob-controller
         class="linked-time"
-        [linkedTime]="linkedTime"
+        [timeSelection]="timeSelection"
```

Comment:
Out of curiosity, did you choose to use timeSelection instead of linkedTimeSelection in this file for consistency with scalars? I'm fine with it but just curious.